src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int VALUE_INDEX = 1;
+    private static final int TYPE_INDEX = 0;
+    private static final String DELIMITER = ",";
+
+    public void getStatistic(String fromFileName, String toFileName) throws RuntimeException {
+        int supply = readData(fromFileName, "supply");
+        int buy = readData(fromFileName, "buy");
+        String report = createReport(supply, buy);
```

Comment:
```suggestion
        String report = createReport(dataFromFile);
```
parse data and calculate it there